### PR TITLE
fix(kafka and derivatives): add `alias` config to avoid clashes with same topic

### PR DESCRIPTION
**Pull request by thalesmg**
Fixes https://emqx.atlassian.net/browse/EMQX-12592
Fixes https://emqx.atlassian.net/browse/EMQX-12585

Release version: e5.7.2

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [x] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

**Comment by thalesmg** [comment]
Depends on https://github.com/kafka4beam/wolff/pull/68 being tagged with 1.10.6

File: apps/emqx_bridge_kafka/src/emqx_bridge_kafka_impl_producer.erl [hunk 1]
```diff
@@ -324,6 +324,12 @@ on_query(
         }),
         do_send_msg(sync, KafkaMessage, Producers, SyncTimeout)
     catch
+        error:{invalid_partition_count, _Count, _Partitioner} ->
+            ?tp("kafka_producer_invalid_partition_count", #{
+                action_id => MessageTag,
+                query_mode => sync
+            }),
+            {error, {unrecoverable_error, invalid_partition_count}};
```
Comment (zmstone) [inline apps/emqx_bridge_kafka/src/emqx_bridge_kafka_impl_producer.erl:332]:
```suggestion
        error:{invalid_partition_count, Count, _Partitioner} ->
            ?tp("kafka_producer_invalid_partition_count", #{
                action_id => MessageTag,
                query_mode => sync
            }),
            {error, {unrecoverable_error, {invalid_partition_count, Count}}};
```

File: changes/ee/breaking-13327.en.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+The directory path scheme for on-disk Kafka/Confluent/Azure Event Hub buffers has changed.  It now uses the Action name instead of the topic name.
+
+Updating to this version will invalidate (not use) old paths, and will require manual cleanup of the old directories.
```
Comment (zmstone) [inline changes/ee/breaking-13327.en.md:3]: Please provide an example of the old files.
```suggestion
Upgrading to this version will invalidate (not use) old buffer files, and will require manual cleanup of the old directories.
```

File: apps/emqx_bridge_kafka/src/emqx_bridge_kafka_impl_producer.erl (hunk 1 above)
Comment (thalesmg) [reply apps/emqx_bridge_kafka/src/emqx_bridge_kafka_impl_producer.erl]: :checkered_flag:

**Review by zmstone — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-2730)

**Pull request merged**